templates/web/base/report/new/after_photo.html

```diff
@@ -4,6 +4,6 @@
         <li>[% loc('For best results include a close-up and a wide shot') %]</li>
     </ul>
     <ul class="dont">
-        <li>[% loc('Avoid personal information and vehicle number plates') %]</li>
+        <li id="photo-tips-avoid-personal-info">[% loc('Avoid personal information and vehicle number plates') %]</li>
```

Comment:
There's a convention of using a `js-` prefix on IDs/classes that are only used for JS functionality, think that would be good to use here (and the other ones too).